cmd/rpcdaemon/commands/erigon_receipts_test.go

```diff
@@ -0,0 +1,30 @@
+package commands
+
+import (
+	"context"
+	"log"
+	"testing"
+
+	"github.com/ledgerwatch/erigon-lib/kv/kvcache"
+	"github.com/ledgerwatch/erigon/cmd/rpcdaemon/rpcdaemontest"
+	"github.com/ledgerwatch/erigon/eth/filters"
+	"github.com/ledgerwatch/erigon/rpc/rpccfg"
+	"github.com/ledgerwatch/erigon/turbo/snapshotsync"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestErigonGetLatestLogs(t *testing.T) {
+	assert := assert.New(t)
+	m, _, _ := rpcdaemontest.CreateTestSentry(t)
+	br := snapshotsync.NewBlockReaderWithSnapshots(m.BlockSnapshots)
+	stateCache := kvcache.New(kvcache.DefaultCoherentConfig)
+	db := m.DB
+	agg := m.HistoryV3Components()
+	api := NewErigonAPI(NewBaseApi(nil, stateCache, br, agg, false, rpccfg.DefaultEvmCallTimeout), db, nil)
+	logs, err := api.GetLatestLogs(context.Background(), filters.FilterCriteria{}, 1)
```

Comment:
I can see some logs by: 
```
logs, _ := api.GetLogs(ctx, filters.FilterCriteria{FromBlock: big.NewInt(0), ToBlock: big.NewInt(1024)})
	for _, l := range logs {
		fmt.Printf("alex: %v\n", l.BlockNumber)
	}
```

Reply:
Thank you. Really appreciated for your help. I will add it tomorrow.